src/containers/SortingView.tsx

```diff
@@ -198,10 +192,20 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
             readOnly={readOnly}
             plugins={plugins}
             calculationPool={calculationPool}
-            width={W}
-            height={H - headingHeight}
+            width={contentWidth}
+            height={contentHeight}
           />
       </div>
+      <div style={footerStyle}>
```

Comment:
In SortingView, the header has been moved to footer and we make better use of the screen real-estate.